pro/src/pages/Reimbursements/Income/Income.spec.tsx

```diff
@@ -275,6 +275,9 @@ describe('Income::Income', () => {
       }
 
       await waitFor(() => expect(screen.getByText(LABELS.venuesSelectorError)).toBeInTheDocument())
+
+      // It should not attempt to fetch income data if no venues are selected.
+      expect(api.getStatistics).toHaveBeenCalledTimes(1)
```

Comment:
ça devrait pas être pas appelé du tout du coup ?

Reply:
"when the user changes venue selection" - sous-entendu, "après que les données de revenu aient été chargés une première fois" donc oui, cette api a été appelée 1 fois, mais ne devrait pas l'être quand la sélection change de façon à ce qu'aucune venue ne soit sélectionnée (on épargne une requête inutile, et on garde les données précédentes à l'écran).